sphinxcontrib/aafig.py

```diff
@@ -76,7 +76,7 @@ class AafigDirective(images.Image):
     def run(self):
         aafig_options = dict()
         image_attrs = dict()
-        own_options_keys = self.own_option_spec.keys() + ['scale']
+        own_options_keys = list(self.own_option_spec.keys()).append('scale')
```

Comment:
Two issues here, first I don't see why the original line won't work in Python3. It should.
The seconds and more important is `list(self.own_option_spec.keys()).append('scale')` will **always** return `None`.

Reply:
`keys()` on a `dict` in py3 returns type `dict_keys`, which is read-only, and doesn't have an append. Needs to be converted to a list to have an append.  For example:

```
>>> x = {"abc": "efg"}
>>> x.keys() + ['scale']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'dict_keys' and 'list'
```
However, yes it always returns `None` as written, it should be `list(x.keys())+['scale']`. I'll fix it.

Reply:
> `keys()` on a `dict` in py3 returns type `dict_keys`, which is read-only

Right! I missed the `.keys()`. The problem is not that it is read-only, but that it's a generator, and like the error says, it doesn't define a `+` operator with a list.

> However, yes it always returns `None` as written, it should be `list(x.keys())+['scale']`. I'll fix it.

Yeah, perfect, that looks good to me!